packages/@react-spectrum/dialog/test/DialogTrigger.test.js

```diff
@@ -362,4 +352,89 @@ describe('DialogTrigger', function () {
       getByRole('dialog');
     }).toThrow();
   });
+
+  it('dismissable modals can be closed by clicking outside the dialog', async function () {
+    function Test({defaultOpen, onOpenChange}) {
+      return (
+        <Provider theme={theme}>
+          <DialogTrigger type="modal" isDismissable defaultOpen={defaultOpen} onOpenChange={onOpenChange}>
+            <ActionButton>Trigger</ActionButton>
+            <Dialog>contents</Dialog>
+          </DialogTrigger>
+        </Provider>
+      );
+    }
+
+    let onOpenChange = jest.fn();
+    let {getByRole} = render(<Test defaultOpen onOpenChange={onOpenChange} />);
+
+    let dialog = getByRole('dialog');
+    expect(dialog).toBeVisible();
+    await waitForDomChange(); // wait for animation
+
+    fireEvent.mouseUp(document.body);
+    expect(dialog).toBeVisible();
+    expect(onOpenChange).toHaveBeenCalledTimes(1);
+    expect(onOpenChange).toHaveBeenCalledWith(false);
+
+    await waitForDomChange(); // wait for animation
+
+    expect(() => {
+      getByRole('dialog');
+    }).toThrow();
+  });
+
+  it('non dismissable modals cannot be closed by clicking outside the dialog', async function () {
+    function Test({defaultOpen, onOpenChange}) {
+      return (
+        <Provider theme={theme}>
+          <DialogTrigger type="modal" defaultOpen={defaultOpen} onOpenChange={onOpenChange}>
+            <ActionButton>Trigger</ActionButton>
+            <Dialog>contents</Dialog>
+          </DialogTrigger>
+        </Provider>
+      );
+    }
+
+    let onOpenChange = jest.fn();
+    let {getByRole} = render(<Test defaultOpen onOpenChange={onOpenChange} />);
+    
+    let dialog = getByRole('dialog');
+    expect(dialog).toBeVisible();
+    await waitForDomChange(); // wait for animation
+
+    fireEvent.mouseUp(document.body);
+    expect(dialog).toBeVisible();
+    expect(onOpenChange).toHaveBeenCalledTimes(0);
+  });
+
+  it('non-modals can be closed by clicking outside the dialog regardless of isDismissable', async function () {
+    function Test({defaultOpen, onOpenChange}) {
+      return (
+        <Provider theme={theme}>
+          <DialogTrigger type="popover" defaultOpen={defaultOpen} onOpenChange={onOpenChange}>
```

Comment:
Shouldn't this example have isDismissable on it?

Reply:
Oh ya lemme add isDismissable=false here